### PR TITLE
Error keys

### PR DESCRIPTION
**Pull request by fefrain**

### CODE REVIEW

File: common/i18n/resources/en.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ module.exports = {
     no: 'No',
     noItems: `$t(common.no) $t(common.item,{'count':2})`,
     of: 'of',
-    required: 'Required',
+    valueRequired: 'Required',
```
Comment (minotogna) [inline common/i18n/resources/en.js]: why has the key changed? if the text says 'Required' I would call the key 'required' . note the keys in common are alphabetically ordered . also, is this still used?

File: common/i18n/resources/en.js [hunk 2]
```diff
@@ -285,32 +276,77 @@ module.exports = {
...
+      valueRequired: 'Required value',
+    },
+
+    surveyInfoEdit: {
+      langRequired: 'Language is required',
+      srsRequired: 'Spatial Reference System is required',
+    },
+
+    taxonomyEdit: {
+      codeDuplicate: 'Duplicate code {{code}}; $t(jobErrors.duplicateRows)',
```
Comment (minotogna) [inline common/i18n/resources/en.js]: why do we still have jobErrors?

File: webapp/utils/validationUtils.js [hunk 3]
```diff
@@ -3,42 +3,53 @@ import * as R from 'ramda'
...
+  }
+}
 
-const getErrorText = (error, i18n, errorKeyPrefix = 'formErrors') => error.key === 'custom'
-  ? error.messages[i18n.lang]
-  : i18n.t([`${errorKeyPrefix}.${error.key}`, error.key], error.params)
-
-const getFieldError = (i18n, errorKeyPrefix) => field => R.pipe(
-  R.pathOr([], [field, 'errors']),
+const getFieldError = (i18n, errorKeyPrefix = null) => field => R.pipe(
```
Comment (minotogna) [inline webapp/utils/validationUtils.js]: do we still need errorKeyPrefix?

File: common/i18n/resources/en.js (hunk 1 above)
Comment (fefrain) [reply common/i18n/resources/en.js]: right

File: webapp/utils/validationUtils.js (hunk 3 above)
Comment (fefrain) [reply webapp/utils/validationUtils.js]: no, I'll remove it

File: common/i18n/resources/en.js (hunk 2 above)
Comment (fefrain) [reply common/i18n/resources/en.js]: no need for them

**Pull request merged**